beater/sipcmbeat.go

```diff
@@ -620,7 +620,9 @@ func (bt *Sipcmbeat) publishEv(srcEv *calltr.EventData) {
 	addFields(event.Fields, "dbg.last_status", ed.LastStatus)
 	addFields(event.Fields, "dbg.msg_trace", ed.LastMsgs.String())
 
-	addFields(event.Fields, "encrypt", strconv.Itoa(int(fFlags))+"|"+bt.validator.Compute())
+	if bt.Config.UseIPAnonymization() {
+		addFields(event.Fields, "encrypt", strconv.Itoa(int(fFlags))+"|"+bt.validator.Compute())
+	}
```

Comment:
I would add an encrypt: 0 field for the unencrypted case and change the if condition to if fFlags != 0 (well, fFlags should now be better named encFlags).